### PR TITLE
FEATURE: Add welcome message for admins.

### PR DESCRIPTION
**Pull request by vinothkannans**

### CODE REVIEW

File: app/models/concerns/roleable.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def regular?
 
   def grant_moderation!
     set_permission('moderator', true)
-    enqueue_welcome_moderator_message
+    enqueue_staff_welcome_message('moderator')
```
Comment (eviltrout) [inline app/models/concerns/roleable.rb]: Why the choice of strings over symbols here? `:moderator`

Comment (vinothkannans) [reply app/models/concerns/roleable.rb]: No good reason. I updated it with symbols now.

**Comment by coding-horror** [comment]
Am I clear to edit copy on this one?

**Comment by SamSaffron** [comment]
@coding-horror once it is merged, sure. 

@vinothkannans  feel free to merge once you address Robin's concern

**Comment by vinothkannans** [comment]
@coding-horror now it's merged.

**Pull request merged**